src/server/routes/routes.ts

```diff
@@ -0,0 +1,189 @@
+import fs from 'fs';
+import { Router, Request } from 'express';
+import type { IncomingHttpHeaders } from 'http';
+
+import getTemplateData from '../data-access';
+import ServiceNames from '../../common/service-names';
+import renderTemplate from '../render-template';
+import { processOrientationOption } from '../../browser/helpers';
+import generatePdf, { previewPdf } from '../../browser';
+import { SendingFailedError, PDFNotFoundError } from '../errors';
+import config from '../../common/config';
+import { OPTIONS_HEADER_NAME } from '../../common/consts';
+import { PreviewReqQuery } from '../../common/types';
+
+const PORT = config.webPort;
```

Comment:
Wouldn't it be better to consume these from env variables?

Reply:
In prod, these variables are set by clowder at runtime :( So no env variables I am afraid.

Reply:
It takes the variables from a file and propagates them through a dependency ('app-common-js'). We could then slap the vars into env, but the issue is that some variables (like other services endpoints) are not strings or different primitive types. So we would have to parse it anyways. And I'd like to do it once after the service startup, rather then every time we want to use a variable.